### ISSUE TITLE
question in ROS2_chicken_model

### ISSUE THREAD

**Issue by Zseawing** [Rak-r/Game_Theory#1]
Hello sir:
in the ROS2_chicken_model, i have 3 question about the chicken_node and ChickenSpeedModulatorNode , the yolov8_msgs.msg and nav2 and pod2 is your  custom packet? I can‘t call him .
![8fa6c371d6679e2333d029ae7f88c11](https://github.com/user-attachments/assets/7dcf97c6-7fce-4d75-88b2-8095920fd3f7)
![c1a463a9177d5a84e1581d17c0edc2f](https://github.com/user-attachments/assets/3333b2a4-4e14-4fdb-9610-8aa0ba0705e3)

**Comment by Rak-r** [comment]
Yes. It is the full custom ros2 stack includes the vehicle Openpodcar_v2 and perception module yolov8 wrapper for ros2.

Pod2_msgs is a custom ros2 message developed for this experiment and demonstration.

If you want to have a full step by step overview I suggest look here: https://github.com/Rak-r/OpenPodCar_V2

The above link contains the sub modules for yolo and game theory.

**Comment by Zseawing** [comment]
thanks,I'm learning about game theory in AV, thank you very much and I am applying to add you on linkedin, hoping for your approval
